### ISSUE TITLE
Neovim startup is much slow in mac than in linux. 

### ISSUE THREAD

**Issue by finviman** [lewis6991/impatient.nvim#73]
I use neovim 0.8, exactly the same config files in both OS. it’ll take 60ms to startup in linux but 180ms in macos m1 pro.
below is the part of --startuptime output , the difference lines:
this is mac:
```
103.941  087.375  087.375: sourcing ~/.config/nvim/init.lua
183.011  044.137: VimEnter autocommands
```
this is linux:
```
030.694  026.166  026.166: sourcing ~/.config/nvim/init.lua
059.600  010.673: VimEnter autocommands
```
I don’t know why, mac hardware is much better than linux, and no harddisk encrypt.

**Comment by lewis6991** [comment]
There's no information here I can use to assist.

**Comment by lewis6991** [comment]
I'm going to close this.

If you can provide any more information then I'm happy to try and help.

**Comment by clason** [comment]
Slow(er) startup due to filesystem access on macOS is just a fact of life that can be observed even without impatient.nvim.

(If I had to speculate, I'd suspect an extra system protection layer.)

**Comment by finviman** [comment]
> I'm going to close this.
> 
> If you can provide any more information then I'm happy to try and help.

Thank you for attention. I fixed it yestoday.
I started alacritty in M1 pro with rosetta translation(it's not neccessary and is not recommended.),  so homebrew install every packages as if it's in Intel platform. This affects performance significantly.
So I reinstall homebrew and all the packages included neovim, now the startup time is under 60ms.

**Comment by FinallyFinancialFreedom** [comment]
> I'm going to close this.
> 
> If you can provide any more information then I'm happy to try and help.

Thank you for attention. I fixed it yestoday.
I started alacritty in M1 pro with rosetta translation(it's not neccessary and is not recommended.),  so homebrew install every packages as if it's in Intel platform. This affects performance significantly.
So I reinstall homebrew and all the packages included neovim, now the startup time is under 60ms. 
I started alacritty in M1 pro with rosetta translation(it's not neccessary and is not recommended.),  so homebrew install every packages as if it's in Intel platform. This affects performance significantly.
So I reinstall homebrew and all the packages included neovim, now the startup time is under 60ms.
